Task_20/DriverFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace Yandex
+{
+    public static class DriverFactory
+    {
+        static IWebDriver _driver = null;
```

Comment:
I would recommend you to try avoiding static drivers using. It may cause issues during parallel tests running.

Reply:
done non-static